src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
```

Comment:
I'm not changing a public API, but add more operations for existing model (gateway in our case), and returning more properties of the existing model as we extend existing model from one node gateway to multi-nodes gateway(with high availability feature). Most customer will still use one node gateway as before if they don't extend their gateway to multi-nodes, and even they extend their gateway to multi nodes, previous public APIs are still usefully, they just not get extra properties and extra nodes' info introduced by HA feature.

The other thing is code in Microsoft.Azure.Management.DataFactories are generated by Hydra code which we won't touch much later on since we are targeting to have ADF V2 product (which is introducing new app model and uses swagger.). And per Mark Cowlishaw, Azure SDK won't accept hydra generated code for new feature, we cannot mix swagger and hydra generated code in our ADF V1 product.